### PR TITLE
new statuten

### PR DESCRIPTION
**Pull request by annatraussnig**
Screenshot 2021-03-12 at 15 19 41

Screenshot 2021-03-12 at 15 19 19

### CODE REVIEW

File: components/Frame/Footer.js [hunk 1]
```diff
@@ -235,7 +235,7 @@ class Footer extends Component {
                 <a>{t('footer/legal/privacy')}</a>
               </LazyLink>
               <br />
-              <LazyLink route='legal/statute'>
+              <LazyLink route='statuten'>
                 <a>{t('footer/legal/statute')}</a>
               </LazyLink>
```
Comment (tpreusse) [inline components/Frame/Footer.js:240]: `statuten` is a path and not a route(name):

```suggestion
              <HrefLink href='/statuten' passHref>
                <a>{t('footer/legal/statute')}</a>
              </HrefLink>
```

File: components/Pledge/LegalOverlay.js [hunk 2]
```diff
@@ -56,8 +76,10 @@ const LegalOverlay = ({ onClose, href, title }) => {
         />
       </OverlayToolbar>
       <OverlayBody>
-        {page ? (
+        {page && page.content ? (
           <page.content />
+        ) : data && data.article ? (
+          <RenderArticle article={data.article} />
```
Comment (tpreusse) [inline components/Pledge/LegalOverlay.js:82]: let's add a loader here and pass it error and loading state

**Review by tpreusse — COMMENTED** [review]
Awesome, thank you. Generally, looks good. Two small change requests.

File: components/Pledge/LegalOverlay.js [hunk 3]
```diff
@@ -13,6 +13,12 @@ import {
 
 import { MdClose } from 'react-icons/md'
 import Loader from '../Loader'
+import { compose, graphql } from 'react-apollo'
+import { getDocument } from '../Article/graphql/getDocument'
+import { splitByTitle } from '../../lib/utils/mdast'
+import { renderMdast } from 'mdast-react-render'
+import createPageSchema from '@project-r/styleguide/lib/templates/Page'
+import Campaign from '../Access/Campaigns/Campaign'
```
Comment (tpreusse) [inline components/Pledge/LegalOverlay.js:21]: your editor went rough

**Review by tpreusse — APPROVED** [review]
LGTM (I didn't test it)

File: components/Pledge/LegalOverlay.js (hunk 3 above)
Comment (annatraussnig) [reply components/Pledge/LegalOverlay.js:21]: :D I copy pasted the loader from somewhere

**Pull request merged**